### PR TITLE
Add autocorrect-py

### PR DESCRIPTION
**Pull request by messense**
If adding a python package is desired, I can continue to work on adding CI/CD for it.

### CODE REVIEW

File: .github/workflows/python.yml [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+      - uses: actions/download-artifact@v2
+        with:
+          name: wheels
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.10
+      - name: Publish to PyPI
+        env:
+          TWINE_USERNAME: __token__
+          TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
```
Comment (messense) [inline .github/workflows/python.yml:176]: FYI, [How can I use API tokens to authenticate with PyPI](https://pypi.org/help/#apitoken)

Comment (huacnlee) [reply .github/workflows/python.yml:176]: I have added PYPI_TOKEN in this repo secrets.

File: autocorrect-py/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+edition = "2018"
+homepage = "https://github.com/huacnlee/autocorrect"
+license = "MIT"
+readme = "../README.md"
+repository = "https://github.com/huacnlee/autocorrect"
+version = "2.1.1"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[lib]
+name = "autocorrect_py"
```
Comment (huacnlee) [inline autocorrect-py/Cargo.toml:13]: Use `autocorrect-py` as Pypi name.

Comment (messense) [reply autocorrect-py/Cargo.toml:13]: See also https://pyo3.rs/v0.17.2/getting_started.html#cargotoml

**Comment by huacnlee** [comment]
https://pypi.org/project/autocorrect-py/

**Comment by messense** [comment]
Thanks for fixing up release CI, YAML is the worst.

**Pull request merged**